analyzers/tests/SonarAnalyzer.Test/TestCases/UnnecessaryBitwiseOperation.CSharp9.cs

```diff
@@ -62,4 +63,53 @@ await foreach (var v in values)
         }
         unchanged = 1;
     }
+
+    public void TupleAssignment()
+    {
+        var i = 0;
+        (i, _) = (1, 1);
+        _ = i | 0x80;
+    }
+
+    public void InterlockedMethods()
```

Comment:
I would structure the test method slightly differently. 
What we are testing here is not much the `Interlocked` methods as such, but rather the behavior in the presence of methods that contain `ref` parameters with names matching the identifier under analysis. `Interlocked` methods are just one example of that, but the same test could be done with a made up method `MethodWithRefParam(ref int i) {}`.

So, in order to avoid red herring for the reader of the test cases, I would name the test method something like `WithRefParameters` and:
- either use my own `MethodWithRefParam(ref int i) {}` to test, and maybe still keep some examples of `Interlocked`, if you find them valuable
- or keep `Interlocked` as a local function, i.e. as a group of examples of the `WithRefParameters` class of test cases